jerry-core/ecma/base/ecma-helpers.h

```diff
@@ -96,6 +96,15 @@ typedef enum
   ECMA_STRING_FLAG_MUST_BE_FREED = (1 << 3), /**< The returned buffer must be freed */
 } ecma_string_flag_t;
 
+/**
+ * Status flags for allowing underscore in number create
+ */
+typedef enum
+{
+  PREFIX_NO_UNDERSCORE = 0, /**< underscore character is not allowed */
+  PREFIX_ALLOW_UNDERSCORE = (1 << 0), /**< underscore character is allowed */
```

Comment:
No need this flag if we have `ECMA_CONVERSION_ALLOW_UNDERSCORE`. Maybe just move the definition here:

```
typedef enum
{
  ECMA_CONVERSION_NO_OPTIONS = 0, /**< no options */
  ECMA_CONVERSION_IGNORE_UNDERSCORE = (1 << 0), /**< ignore underscore characters */
}
```
And call it ignore underscore.